README.md

```diff
@@ -34,6 +39,31 @@ It is possible to send args to the script under test by adding -- after the load
 ./bin/load-runner.js -s ./scripts/dummy.js -c 5 -n 100 -r 10 -o -- -testarg1 testval1
 ```
 
+# Environment variables
+
+There are some environment variables generated for the test script that can be useful. You can use Node.js `process.env` object to access them.
+
+* `LR_FLOW_NUMBER` - number generated each run, value depends on percentages specified by `-f` parameter, see [Flow numbers](#flow-numbers)  
+* `LR_TOTAL_RUNS` - total number of runs, it's value of `-n` parameter
+* `LR_RUN_NUMBER` - current test run number
+* `LR_RAND` - randomly generated value for each run, float between 0 and 1, based on random generation with `--seed` parameter
+
+Look at [scripts/dummy_env_vars.js](scripts/dummy_env_vars.js) for example of using this.
+
+
+# Flow numbers
+
+Flow number functionality can be used inside test script to determine which branch of script to be executed. 
+It's specified by `-f` parameter and can be accessed by `LR_FLOW_NUMBER` environment variable in the script.
+
+For example this executes 5 times flow 0, 3 times flow 1, 2 times flow 2 from total of 10 runs.
+
+```
+./bin/load-runner.js -s ./scripts/dummy_flow_numbers.js -n 10 -o -f 50 30 20 --seed 123 
+```
+
+Flow numbers are randomly shuffled, but it's possible to use `--seed` parameter for repeatable result. 
```

Comment:
For load-runner, i'd see this as being a pattern passed in as an arg.
e.g. if the pattern is passed in as a string "AAB", 6 test runs would look like this
- pass A to the test script
- pass A to the test script
- pass B to the test script
- pass A to the test script
- pass A to the test script
- pass B to the test script
i.e. it would have a 66.6:33.3 ratio of A's to B's in a repeating pattern.
If the pattern was "XXYZ", the ratio would be 50:25:25, and repeating in a pattern of X, X, Y, Z.


If you do this in the test script instead of load runner, the test script would need to know a few things:
- the pattern (ratio & number of paths) (can be kept in the test script)
- the current test number (passed in from load-runner as an arg)
The test script would then decide based on those values if it should go down path A or B.
e.g. if we want a pattern of 75% A's and 25% B's (AAAB), the path can be calculated like this:
```js
if ([1,2,3].indexOf(testNumber % 4)) {
  path = 'A'
} else {
  path = 'B'
}
```